app/assets/stylesheets/application.scss

```diff
@@ -20,3 +20,58 @@
 .top-bar .name {
   padding-right: 15px;
 }
+
+@mixin vertical-align {
+  position: relative;
+  top: 50%;
+  -webkit-transform: translateY(-50%);
+  -ms-transform: translateY(-50%);
+  transform: translateY(-50%);
+}
+
+
+.leaf {
+  background: url('http://static.pexels.com/wp-content/uploads/2014/07/beach-coast-holiday-2190-825x550.jpg') no-repeat center center;
```

Comment:
Prefer double-quoted strings<br>URL `http://static.pexels.com/wp-content/uploads/2014/07/beach-coast-holiday-2190-825x550.jpg` should not contain protocol or domain